### PR TITLE
Use master transactions as query leases

### PR DESCRIPTION
**Pull request by gritukan**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: yt/python/yt/environment/init_query_tracker_state.py [hunk 1]
```diff
@@ -393,6 +393,89 @@
...
+            optimize_for="lookup",
+            attributes={
+                "tablet_cell_bundle": SYS_BUNDLE_NAME,
+            },
+        ),
+        use_default_mapper=True,
+    ),
+    Conversion(
+        "finished_queries",
+        table_info=TableInfo(
```
Comment (Krisha11) [inline yt/python/yt/environment/init_query_tracker_state.py:432]: What changes in this and the following conversions?

Comment (gritukan) [reply yt/python/yt/environment/init_query_tracker_state.py:432]: It adds a new field lease_transaction_id and removes field ping_time

**Comment by Krisha11** [comment]
LGTM - ready to merge after fixing conflict/codestyle

File: yt/python/yt/environment/init_query_tracker_state.py (hunk 1 above)
Comment (gritukan) [reply yt/python/yt/environment/init_query_tracker_state.py]: Indeed, fixed.

**Review by Krisha11 — APPROVED** [review]

**Pull request closed without merging**